src/modules/generators/fractals/fbm.rs

```diff
@@ -0,0 +1,163 @@
+// Copyright 2016 The Noise-rs Developers.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use num_traits::Float;
+use math;
+use math::{Point2, Point3, Point4};
+use NoiseModule;
+use modules::Perlin;
+use super::util::*;
+
+/// Default noise seed for the Fbm noise module.
+pub const DEFAULT_FBM_SEED: usize = 0;
+/// Default number of octaves for the Fbm noise module.
+pub const DEFAULT_FBM_OCTAVE_COUNT: usize = 6;
+/// Default frequency for the Fbm noise module.
+pub const DEFAULT_FBM_FREQUENCY: f32 = 1.0;
+/// Default lacunarity for the Fbm noise module.
+pub const DEFAULT_FBM_LACUNARITY: f32 = 1.91803398875;
+/// Default persistence for the Fbm noise module.
+pub const DEFAULT_FBM_PERSISTENCE: f32 = 0.5;
+/// Maximum number of octaves for the Fbm noise module.
+pub const FBM_MAX_OCTAVES: usize = 32;
+
+/// Noise module that outputs classical fBm noise.
```

Comment:
Certainly. I'll try to fill out the description further, but for a quick note, here's the wikipedia article on fBm: https://en.wikipedia.org/wiki/Fractional_Brownian_motion